tests/integration/data_sources_and_expectations/test_misconfigured_expectations.py

```diff
@@ -0,0 +1,184 @@
+import datetime as dt
+
+import pandas as pd
+import pytest
+
+import great_expectations.expectations as gxe
+from tests.integration.conftest import parameterize_batch_for_data_sources
+from tests.integration.data_sources_and_expectations.test_canonical_expectations import (
+    ALL_DATA_SOURCES,
+)
+from tests.integration.test_utils.data_source_config.base import DataSourceTestConfig
+
+PANDAS_DATA_SOURCES: list[DataSourceTestConfig] = [
+    ds for ds in ALL_DATA_SOURCES if ds.label.startswith("pandas")
+]
+SPARK_DATA_SOURCES: list[DataSourceTestConfig] = [
+    ds for ds in ALL_DATA_SOURCES if ds.label.startswith("spark")
+]
+SQL_DATA_SOURCES: list[DataSourceTestConfig] = [
+    ds for ds in ALL_DATA_SOURCES if ds not in PANDAS_DATA_SOURCES and ds not in SPARK_DATA_SOURCES
+]
```

Comment:
I don't love this (or the test below) - is there a better way to do this?

Should we group ds test configs based on Pandas/Spark/SQL based on an enum? Should we have a file that is the source of truth for the test configs (and their appropriate groupings)?

Reply:
We now have `SQL_DATA_SOURCES` defined in `test_canonical_expectations` (I plan to move it somewhere better, but that's where it is now).